compiler/src/dotty/tools/dotc/core/Flags.scala

```diff
@@ -542,6 +542,9 @@ object Flags {
   /** Either method or lazy */
   final val MethodOrLazy = Method | Lazy
 
+  /** Either method or module */
+  final val MethodOrModule = Method | Module
```

Comment:
Do we really need to define a new one for only one use?

Reply:
It's  takes slightly more efficient that way. The `|` computation takes time, it's not just a simple bitwise `|`.